ballista/rust/core/src/utils.rs

```diff
@@ -151,7 +151,7 @@ fn build_exec_plan_diagram(
     id: &mut AtomicUsize,
     draw_entity: bool,
 ) -> Result<usize> {
-    let operator_str = if plan.as_any().downcast_ref::<HashAggregateExec>().is_some() {
+    let operator_str = if plan.as_any().downcast_ref::<AggregateExec>().is_some() {
         "HashAggregateExec"
```

Comment:
Should we also change the operator string here to remove Hash ?

Reply:
Thanks @andygrove. I've updated all these explanation strings and checked all occurrences of `hash_aggregate` and `HashAggregate`.